pkgs/applications/misc/furtherance/default.nix

```
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "furtherance";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "lakoliu";
+    repo = "Furtherance";
+    rev = "v${version}";
+    sha256 = "oIzwFPJUY+OwT2B6n7NPg8bMxEd4D0MKPe3GQxrXmEU=";
+  };
+
+  patches = [
+    ./patches/furtherance-gnome-post-install.patch
```

Please move/rename the patch:
```suggestion
    ./gnome-post-install.patch
```

---

done!